README.md

```diff
@@ -1 +1,79 @@
-# Capstone-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Educatia Software Development Website
+
+> This project is the final capstone for the first module (HTML - CSS).
+
+The next website is about a Online Bootcamp about Software Development, it's just an instance according to the website to sell a Software Development.
+For this project, I've used HTML, CSS, and Bootstrap to the entire website, also I've included GRID and flexbox to make more professional distribution of elements inside specific parts of the website.
+
+## Desktop 💻
+<img src="img/desktop.png" alt="DesktopLayout" width="100%">
+
+## Mobile 📱
+<img src="img/mobile.png" alt="DesktopLayout" width="50%">
+
+
+## Built With 🔨
+
+- HTML5
+- CSS
+- Bootstrap
+- Visual Studio Code
+- Git & Github
+
+## Live Demo 👀
```

Comment:
- [ ] As requested by the former code reviewer, which is a requirement for this project, please kindly a link to the video  presenting the features of the project, the maximum length of the video is 5 minutes, the video contains a clear and objective description about the project in professional English, the video contains a demo of the project, showing the pages in different screen sizes, the video contains a highlight about a piece of the code the student finds interesting or is proud of making.